pkg/serviceapi/handler.go

```diff
@@ -21,7 +21,7 @@ func (h serviceHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if contentType != "" && contentType != "application/json" {
 		logrus.Errorf("%s is not a supported Content-Type", r.Header.Get("Content-Type"))
 	}
-
+	r.ParseForm()
```

Comment:
Should at least log error if this fails, maybe even bail back to client with 500